### ISSUE TITLE
Incorrect help message for boolean options with arity 1

### ISSUE THREAD

**Issue by prewersk** [remkop/picocli#612]
If we print help for the following example command

```
@Command(mixinStandardHelpOptions = true)
public class ExampleCommand {

  @Option(names = { "-b" }, arity = "1")
  boolean booleanWithArity1;

}
```

the message looks like this

```
Usage: <main class> [-hVb]
  -b=<booleanWithArity1>
  -h, --help                Show this help message and exit.
  -V, --version             Print version information and exit.
```

As we expect value for the '-b' option, I think that usage line should look like

```
Usage: <main class> [-hV] [-b=<booleanWithArity1>]
```

The problem does not occur when we define only a name with double '-' for such an option.

```
@Command(mixinStandardHelpOptions = true)
public class ExampleCommand {

  @Option(names = { "--b" }, arity = "1")
  boolean booleanWithArity1;

}
```

```
Usage: <main class> [-hV] [--b=<booleanWithArity1>]
      --b=<booleanWithArity1>

  -h, --help      Show this help message and exit.
  -V, --version   Print version information and exit.
```

**Comment by remkop** [comment]
Thank you for the bug report! Looking into this.

**Comment by remkop** [comment]
Fixed in master. This will be included in the upcoming 3.9.2 release.
Please verify.

**Comment by prewersk** [comment]
Great, thanks for fixing the bug so quickly.

sob., 19 sty 2019 o 11:31 Remko Popma <notifications@github.com> napisał(a):
